assets/scripts/opt-out.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+(function ($) {
+  // Triggers out-out cookie creation when click on link.
+  $('.core-standards-optout-link').on('click', function(e) {
+    e.preventDefault();
+    gaOptout()
+    alert(core_standards_optout.confirmation_message);
```

Comment:
The part before the dot should be the namespace of the plugin; i.e., only `core_standards` and the `optout` part needs to move after the period.